tempesta_fw/http_sess.c

```diff
@@ -83,10 +84,10 @@ static TfwCfgSticky tfw_cfg_sticky;
 static struct crypto_shash *tfw_sticky_shash;
 static char tfw_sticky_key[STICKY_KEY_MAXLEN];
 
-SessHashBucket sess_hash[SESS_HASH_SZ] = {
-	[0 ... (SESS_HASH_SZ - 1)] = {
-		HLIST_HEAD_INIT,
-	}
+static SessHashBucket sess_hash[SESS_HASH_SZ] = {
+//	[0 ... (SESS_HASH_SZ - 1)] = {
+//		HLIST_HEAD_INIT,
+//	}
```

Comment:
Don't use C++ style comments. Don't leave dead (just commented out) code. Why did you remove the initialization? I know that it's just zeroying, but using the initialization is still a good practice. At least use the list initializer in `tfw_http_sess_init()` near to spinlock initialization.